use/auth.js

```diff
@@ -398,10 +399,18 @@ export const useAuth = () => {
       state.error = e
     }
   }
-
+  async function updateTimeZone() {
+    const { zone } = await getAccount()
+    if (!zone) {
+      await updateAccount({
+        zone: Intl.DateTimeFormat().resolvedOptions().timeZone
+      })
+    }
+  }
   async function signUserIn(email, password) {
     try {
       await firebase.auth().signInWithEmailAndPassword(email, password)
+      updateTimeZone()
```

Comment:
Maybe it's better just do it inside setUser method? It will solve Google Sign In too